libs/output-mapping/src/TableLoader.php

```diff
@@ -215,12 +215,24 @@ private function createTable(MappingDestination $destination, array $columns, ar
         $headerCsvFile = new CsvFile($tmp->createFile($destination->getTableName().'.header.csv')->getPathname());
         $headerCsvFile->writeRow($columns);
 
-        $this->clientWrapper->getTableAndFileStorageClient()->createTableAsync(
-            $destination->getBucketId(),
-            $destination->getTableName(),
-            $headerCsvFile,
-            $loadOptions,
-        );
+        try {
+            $this->clientWrapper->getTableAndFileStorageClient()->createTableAsync(
+                $destination->getBucketId(),
+                $destination->getTableName(),
+                $headerCsvFile,
+                $loadOptions,
+            );
+        } catch (ClientException $e) {
```

Comment:
Pro zakladani table definition jsme to meli. Pro zakladani netypovane tabulky ne a koncilo to app errorem